src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadocInlineReturn.java

```diff
@@ -0,0 +1,45 @@
+/*
+SummaryJavadoc
+violateExecutionOnNonTightHtml = (default)false
+forbiddenSummaryFragments = (default)^$
+period = (default).
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.summaryjavadoc;
+
+class InputSummaryJavadocInlineReturn {
+    /**
+     * {@return a zero, note that no period is ok as it is set by javadoc tool}
+     */
+    private int returnFoo1() // ok
+    {
+        return 0;
+    }
+
+    /**
+     * {@return a {@code 0}}
+     */
+    private int returnFoo2() // ok
+    {
+        return 0;
+    }
+
+    // violation below
+    /**
+     * {@return}
```

Comment:
Is this violation because the return is completely empty?

Reply:
Yes, ergo there is no summary.

Reply:
It might good to add reason to comment 
`// violation below, because the return is completely empty`

Better to be very clear in tests.

Reply:
Done.